ui/app/templates/components/license-info.hbs

```diff
@@ -54,7 +54,7 @@
       <div class="field box is-fullwidth is-shadowless is-paddingless is-marginless">
         <InfoTableRow @label="License ID" @value={{licenseId}} />
         <InfoTableRow @label="Valid from" @value={{startTime}}>
-          {{date-format model.startTime 'MMM DD, YYYY hh:mm:ss A'}} to {{date-format expirationTime 'MMM DD, YYYY hh:mm:ss A'}}
+          {{date-format startTime 'MMM dd, yyyy hh:mm:ss a'}} to {{date-format expirationTime 'MMM dd, yyyy hh:mm:ss a'}}
```

Comment:
Just double checking that removing `model` intentionally here

Reply:
yes, this was strange.  It threw an error, and I double-checked and the param is passed into the component as @startTime = {{model.startTime}}.